fluidreview/views_test.py

```diff
@@ -42,8 +43,24 @@ def test_webhook_fail_auth(client, settings, token_missing, mocker):  # pylint:
         headers['HTTP_AUTHORIZATION'] = 'Basic abc'
 
     url = reverse('fluidreview-webhook')
-    resp = client.post(url, data='body', content_type='text/plain', **headers)
-    assert resp.status_code == status.HTTP_403_FORBIDDEN
+    with pytest.raises(FluidReviewException):
+        client.post(url, data='body', content_type='text/plain', **headers)
+    assert WebhookRequest.objects.count() == 0
+
+
+def test_webhook_with_accept_header(client, settings, mocker):  # pylint: disable=unused-argument
+    """
+    If the accept header contains an unsupported type, raise an exception
+    """
+    mocker.patch('fluidreview.api.FluidReviewAPI')
+    settings.FLUIDREVIEW_WEBHOOK_AUTH_TOKEN = 'xyz'
+    headers = {
+        'HTTP_ACCEPT': 'video/mp4',
+        'HTTP_AUTHORIZATION': 'Basic xyz'
+    }
+    url = reverse('fluidreview-webhook')
+    with pytest.raises(FluidReviewException):
```

Comment:
Can you assert the content of the exception, at least enough of it to verify that it's the accept header causing the problem and not the auth token?

Reply:
I'll add it to the test above it (authentication failures), because once the `Accept` headers are ignored this won't throw exceptions anymore.